### PR TITLE
Second 2210 release with re-worked Settings

### PR DESCRIPTION
**Pull request by aprantl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+in the properties file.
+
+* Configuration has been refactored so that the GlobalLink settings are not 
+automatically published with the pages that they are linked to. 
+
+  **Upgrade Steps:**
+  1. Create new folders for your global- and site-specific Settings at  
+  `/Settings/Options/Settings/Translation Services` and
+  `<SITE_ROOT>/Options/Settings/Translation Service`
+  2. Move the existing GlobalLink settings into the global- and site-specific
```
Comment (stefanpieper) [inline CHANGELOG.md:25]: State naming convention for settings document explicitly? In apps/studio-client/shared/js/gcc/src/GccWorkflowPlugin.ts it looks like the settings document's name is fixed.

Are these site-specific folders required in any case or just, if site-specific GlobalLink settings are used? Is the global settings folder required in any case as soon as the extension is active?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+  **Upgrade Steps:**
+  1. Create new folders for your global- and site-specific Settings at  
+  `/Settings/Options/Settings/Translation Services` and
+  `<SITE_ROOT>/Options/Settings/Translation Service`
+  2. Move the existing GlobalLink settings into the global- and site-specific 
+  folders respectively.
+  3. Remove the links to the GlobalLink settings from the site's pages and publish
+  these pages.
+  4. De-publish the GlobalLink settings and make sure that there are no links to
+  them anymore by checking the system tab. In the future, you don't need to
```
Comment (stefanpieper) [inline CHANGELOG.md:30]: "don't need to" -> "should not" (in order not to potentially expose sensitive data).

File: docs/administration.md [hunk 3]
```diff
@@ -107,17 +107,23 @@ next chapter, but it is not recommended. The same applies to the parameters
...
-names and values to enable site-specific connections. However, global settings 
-like `dayOffsetForDueDate` have to be defined in `/Settings/Options/Settings/GlobalLink`.
-
-Therefore, you need to add a struct to the GlobalLink settings, named
-`globalLink`. **Make sure to restrict read and write rights of this content to 
-those user groups that actually need access to prevent leaking sensitive information.**
-
-Within that struct the following parameters must/can be specified:
+GlobalLink Settings can be configured globally for all sites or specifically 
+for some site. The Settings can be located in the following folders:
```
Comment (stefanpieper) [inline docs/administration.md:111]: "or specifically for some site." -> "or specifically for some site. Site-specific settings override global settings except for `dayOffsetForDueDate`."

File: docs/administration.md [hunk 4]
```diff
@@ -132,14 +138,17 @@ Within that struct the following parameters must/can be specified:
...
-    Window to lie within the future in days. 
+    Window to lie within the future in days.
     (_optional_, default: `0`, type:`Integer`, scope:**global**)
 * `retryCommunicationErrors` Number of retries in case of a communication error
     with GlobalLink. (_optional_, default: `5`, type:`Integer`)
 * `isSendSubmitter` Defines if the name of the editor that started the workflow 
     is send to GlobalLink as part of the submission.
     (_optional_, default: `false`, type:`Boolean`)
 
+Be aware the `dayOffsetForDueDate` can only be configured in the global
```
Comment (stefanpieper) [inline docs/administration.md:149]: "Be aware the" -> "Be aware that the"

File: docs/development.md [hunk 5]
```diff
@@ -149,13 +149,13 @@ to the variable `DEFAULT_WORKFLOWS`.
 ## Patch/Edit Site Homepages
 
 For each master site for which you want to start GlobalLink Translation Workflows
-from, you need to add `/Settings/Options/Settings/GlobalLink` settings document
+from, you need to add `/Settings/Options/Settings/Translation Services/GlobalLink` settings document
 to the linked settings. To patch a homepage in server export you can use
```
Comment (stefanpieper) [inline docs/development.md:153]: Huh? "add [...] settings document to the linked settings"?

File: docs/development.md [hunk 6]
```diff
@@ -149,13 +149,13 @@ to the variable `DEFAULT_WORKFLOWS`.
...
 the following SED command:
 
 ```bash
 sed --in-place --expression \
-  "\\#.*<linkedSettings>.*#a <link href=\"../../../../../../Settings/Options/Settings/GlobalLink.xml\" path=\"/Settings/Options/Settings/GlobalLink\"/>" \
+  "\\#.*<linkedSettings>.*#a <link href=\"../../../../../../Settings/Options/Settings/Translation Services/GlobalLink.xml\" path=\"/Settings/Options/Settings/Translation Services/GlobalLink\"/>" \
```
Comment (stefanpieper) [inline docs/development.md:158]: "Huh?" again.

**Review by stefanpieper — CHANGES_REQUESTED** [review]

File: docs/release/manual-test-steps.md [hunk 7]
```diff
@@ -40,12 +40,10 @@ home folder
 ## Scenario: Happy Path
 
 1. Log in as Rick C.
-2. Open the GlobalLink settings `/Settings/Options/Settings/GlobalLink`
+2. Open the GlobalLink settings `/Settings/Options/Settings/Translation Services/GlobalLink`
     1. _type_ is set to “default”
     2. _dayOffsetForDueDate_ is set to 20
     3. Credentials for gcc are entered (**automatic** workflow key)
-    4. Document _GlobalLink_ is linked in homepage document of site you want to
```
[the remark this answers is not in the document]
Comment (aprantl) [reply docs/release/manual-test-steps.md:47]: You are welcome ;) It wasn't only search and replace.

**Review by stefanpieper — APPROVED** [review]

**Pull request merged**